apps/ledger-live-desktop/src/renderer/components/DeviceAction/index.tsx

```diff
@@ -38,56 +36,39 @@ import {
   renderInstallingLanguage,
 } from "./rendering";
 
-type OwnProps<R, H, P> = {
-  overridesPreferredDeviceModel?: DeviceModelId,
-  Result?: React$ComponentType<P>,
-  onResult?: P => void,
-  onError?: () => void,
-  action: Action<R, H, P>,
-  request: R,
+type Props<R, H, P> = {
+  overridesPreferredDeviceModel?: DeviceModelId;
+  Result?: React.ComponentType<P>;
+  onResult?: (_: P) => void;
+  onError?: () => void;
+  action: Action<R, H, P>;
+  request: R;
+  status?: H;
```

Comment:
`status` can be undefined here, which can cause pb just below when destructuring it

Reply:
I mean, this is a TS mess that we need to take some time to fix. Thanks for doing it, we should try to fix the rest in another PR